src/frontend/components/Chat/SharedChatComponents/ChatPresenceItem/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box, Text } from 'grommet';
+import React from 'react';
+
+import { ChatPresenceType } from 'data/stores/usePresenceItemsStore';
+
+interface ChatMessageItemProps {
+  presenceItem: ChatPresenceType;
+}
+
+export const ChatPresenceItem = ({ presenceItem }: ChatMessageItemProps) => {
+  return (
+    <Box
+      align="center"
+      margin={{
+        horizontal: '20px',
+        vertical: '5px',
+      }}
+    >
+      <Box
+        align="center"
+        border={{
+          color: 'blue-chat',
+          size: 'xsmall',
+        }}
+        margin={{
+          horizontal: '20px',
+        }}
+        pad={{
+          horizontal: '5px',
+          vertical: '2px',
+        }}
+        round="12px"
+      >
+        <Text color="blue-chat" size="10px">
+          {presenceItem.type === 'arrival'
```

Comment:
```suggestion
          {presenceItem.type === presenceType.ARRIVAL
```

Reply:
done in [d3f94d0bd0cc1cf1b62ffffc209e0eb7aef762d0](https://github.com/openfun/marsha/pull/1279/commits/d3f94d0bd0cc1cf1b62ffffc209e0eb7aef762d0)